tests/test_projectbuilder.py

```diff
@@ -520,3 +521,28 @@ def test_metadata_invalid_wheel(tmp_dir, test_bad_wheel_path):
 
     with pytest.raises(ValueError, match='Invalid wheel'):
         builder.metadata_path(tmp_dir)
+
+
+@pytest.fixture()
+def mock_tomli_not_available(mocker):
+    loads = mocker.patch('tomli.loads')
+    tomli = sys.modules['tomli']
+    try:
```

Comment:
Here you want

```python
patch.dict(sys.modules, {'tomli': None})
```

Reply:
Thanks, I did not know about it :stuck_out_tongue: